word_embedding/train_v2.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
```

Comment:
The source code should be renamed into a more meaningful name, by considering that we will add more examples into this directory.